lib/util.js

```diff
@@ -75,3 +75,13 @@ exports.composeSourceMaps = function(formerMap, latterMap) {
 
     return smg.toJSON();
 };
+
+exports.extractInlineSourceMaps = function (source) {
+    var re = /\s*\/\/(?:@|#) sourceMappingURL=data:application\/json;base64,(\S*)$/m,
```

Comment:
this can be improved

Reply:
i know this is a 5-year old PR, but i don't think this will work with URLs with a charset specified, like `data:application/json;charset=utf-8;base64,...`